templates/rke2/overlay/opt/corral/rke2/init-cluster.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+CORRAL_api_host="${CORRAL_fqdn}"
+echo "corral_set api_host=${CORRAL_api_host}"
+
+mkdir -p /etc/rancher/rke2
+cat > /etc/rancher/rke2/config.yaml <<- EOF
+write-kubeconfig-mode: 644
+cni: ${CORRAL_cni}
+tls-san:
+  - ${CORRAL_api_host}
+EOF
+
+# apt install -y jq || true
```

Comment:
Can this be removed?

Reply:
I removed it and moved it to preflight.sh, as ubuntu would need this pack to be able to use jq